sse2neon.h

```diff
@@ -743,6 +744,24 @@ FORCE_INLINE void _mm_storeu_si128(__m128i *p, __m128i a)
     vst1q_s32((int32_t *) p, vreinterpretq_s32_m128i(a));
 }
 
+FORCE_INLINE void _mm_storeu_si64(void *p, __m128i a)
+{
+    int64x1_t lo =  vget_low_s64(vreinterpretq_s64_m128i(a));
+	* (int64x1_t *) p = lo;
```

Comment:
I think it can be replaced with
```
vst1q_lane_s64((int64_t *) p, vreinterpretq_s64_m128i(a), 0);
```

Reply:
This would write a 0 into the second 64bits, whereas the intent is to only write into the first 64 bits. So I don't think the suggested replacement would work

Reply:
As I replaced it with `vst1q_lane_s64`, it does not behave like you described.
Have you tried it before?